content/04-guides/01-database-workflows/08-sql-views.mdx

```diff
@@ -0,0 +1,378 @@
+- Manually update the Prisma schema to include the view as a model.
+- Generate Prisma Client and write a simple Node.js script to read data from the view.
+
+## Prerequisites
+
+In order to follow this guide, you need:
+
+- a [PostgreSQL](https://www.postgresql.org/) database server running
+- the [`createdb`](https://www.postgresql.org/docs/9.1/app-createdb.html) command line utility
+- the [`psql`](https://www.postgresql.org/docs/10/app-psql.html) command line client for PostgreSQL
+- the [Node.js](https://nodejs.org/) runtime for JavaScript installed on your machine
+
+## Limitations
+
+Be aware of the following limitations when using views with Prisma:
+
+- You must manually add each view as a model to the Prisma schema. Introspection does not add views to the schema.
+- Views must include a unique column - such as an ID.
+- The generated Prisma Client will include queries such `create`, `delete`, and `update`, even though you cannot perform these queries on a view.
+
+## 1. Create a new database and project directory
+
+To create a new database:
+
+1. Create a project directory where you can put the files you'll create throughout this guide:
+
+   ```
+   mkdir sql-views-demo
+   cd sql-views-demo
+   ```
+
+2. Next, make sure that your PostgreSQL database server is running. Then execute the following command in your terminal to create a new database called `SqlViews`:
+
+   ```
+   createdb SqlViews
+   ```
+
+3. Validate that the database was created by running the `\dt` command which lists all tables (_relations_) in your database (right now there are none):
+
+   ```
+   psql -d SqlViews -c "\dt"
+   ```
+
+## 2. Create two tables with a foreign key
+
+In this section, you'll **create two tables where one references the other via a foreign key** in the `SqlViews` database.
+
+1. Create a new file called `sql-views-tables.sql` and add the following code to it:
+
+   ```sql
+   CREATE TABLE "public"."User" (
+     id SERIAL PRIMARY KEY,
+     email TEXT UNIQUE
+     name TEXT
+   );
+
+   CREATE TABLE "public"."Post" (
+     id SERIAL PRIMARY KEY,
+     title TEXT,
+     content TEXT,
+     published BOOLEAN,
+     "authorId" INTEGER,
+     CONSTRAINT author FOREIGN KEY ("authorId") REFERENCES "public"."User" (id)
+   );
+   ```
+
+2. Run the SQL statement against your database to create the two tables:
+
+   ```
+   psql SqlViews < sql-views-tables.sql
+   ```
+
+3. Run the following command to validate that the tables were created:
+
+   ```
+   psql -d SqlViews -c "\dt"
+   ```
+
+You just created two tables named `User` and `Post` in the database. The `Post` table references the `User` table via the foreign key defined on the `authorId` column.
+
+## 3. Create a view named 'Draft'
+
+In this section you will create a view named Drafts that returns the post title and author email of all posts that have not been published. To create a view:
```

Comment:
is "returns" the correct word here? does a view really "return" data? (_maybe_ more along the lines of the "the view _represents_ a model that returns the post title and author ....")